sandbox/pages/SandboxPop.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <SandboxLayout
+    :links="inject('app-links', [])"
+    title="KStepper"
+  >
+    <div class="kstepper-sandbox">
+      <SandboxSectionComponent>
+        <iframe
+          allowfullscreen
+          height="450"
+          src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FYze0SWXl5nKjR0rFdilljK%2FComponents%3Ftype%3Ddesign%26node-id%3D1470%253A3448%26mode%3Ddesign%26t%3DJ5otgKUNrASQZmIz-1"
+          style="border: 1px solid rgba(0, 0, 0, 0.1);"
+          width="800"
+        />
```

Comment:
Let's not use an iframe for Figma; since the sandbox is public, this won't be great for anyone that stumbles on the sandbox site. If there are any other references, can you remove them as well?
![image](https://github.com/Kong/kongponents/assets/2229946/88130e4a-9745-44a7-a215-4e838bf936e8)

Reply:
Updated